src/KnxDevice.cpp

```diff
@@ -87,7 +89,7 @@ e_KnxDeviceStatus KnxDevice::begin(HardwareSerial& serial, word physicalAddr) {
     _state = IDLE;
     DEBUG_PRINTLN(F("Init successful"));
     _lastInitTimeMillis = millis();
-    _lastTXTimeMicros = _lastTXTimeMicros = micros();
+    _lastTXTimeMicros = _lastRXTimeMicros = micros();
```

Comment:
This is the "begin". So there is now "last" available. It makes no sense to use lastTX, and it makes no sense to use lastRX. If you want to improve, just use "micros()".

Reply:
Sure, last suggestion makes sense. Just update the code in your branch. As far as I know, this will immediatelly reflect to your PR.